### PR TITLE
Added a step in Makefile to autogenerate the index.html

### PR DESCRIPTION
**Pull request by e-minguez**

### CODE REVIEW

**Review by alknopfler — APPROVED** [review]
LGTM

File: _config.yml
Comment (diconico07) [inline _config.yml:1]: Why are you removing this ? The `charts` directory is not needed for the standard helm repository (it is only used by Rancher if you specify a git chart repository).

File: charts/akri/0.10.4/Chart.yaml [hunk 1]
```diff
@@ -4,3 +4,4 @@ description: A Helm chart for Akri
 name: akri
 type: application
 version: 0.10.4
+icon: https://raw.githubusercontent.com/suse-edge/charts/_images/akrilogo.png
```
Comment (diconico07) [inline charts/akri/0.10.4/Chart.yaml:7]: This won't survive a `make charts`, and I think we should add the icon in the upstream chart (it's in my todo list to do a PR for this)

File: README.md [hunk 2]
```diff
@@ -1 +1,37 @@
...
+| Chart | Description |
+| - | - |
+| [Akri](charts/akri) | A Kubernetes Resource Interface for the Edge |
+
+# Generate index.html
+
+```
+helm plugin install https://github.com/halkeye/helm-repo-html
+# Generate your yaml and html
+helm repo index ./
```
Comment (diconico07) [inline README.md:31]: The `index.yaml` is generated by the `make charts` so no need to rebuild it here.

File: README.md [hunk 3]
```diff
@@ -1 +1,37 @@
...
+helm repo-html -t index.tpl
+# Now save the two file
+git add index.yaml index.html
```
Comment (diconico07) [inline README.md:34]: Same here the `index.yaml` should already be handled when generating the `index.html`

**Review by diconico07 — CHANGES_REQUESTED** [review]
The index.html and the template is great, thanks.

File: _config.yml
Comment (e-minguez) [reply _config.yml:1]: I thought this was for the Jekyll GH pages. I'll fix it, thanks!

File: charts/akri/0.10.4/Chart.yaml (hunk 1 above)
Comment (e-minguez) [reply charts/akri/0.10.4/Chart.yaml:7]: I'll remove it then :) It will use the placeholder meanwhile

**Review by diconico07 — APPROVED** [review]

**Pull request merged**